src/Commands/Modules/UselessModules.php

```diff
@@ -0,0 +1,234 @@
+<?php
+/**
+ * Copyright (c) Since 2020 Friends of Presta
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file docs/licenses/LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to infos@friendsofpresta.org so we can send you a copy immediately.
+ *
+ * @author    Friends of Presta <infos@friendsofpresta.org>
+ * @copyright since 2020 Friends of Presta
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License ("AFL") v. 3.0
+ */
+
+declare(strict_types=1);
+
+namespace FOP\Console\Commands\Modules;
+
+use FOP\Console\Command;
+use Module;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class UselessModules extends Command
+{
+    /**
+     * @var array possible allowed command
+     */
+    private const ALLOWED_COMMAND = ['status', 'uninstall', 'install'];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('fop:modules')
+            ->setDescription('Manage non-essential and stats modules.')
+            ->setHelp('This command Uninstall or Install non-essential and stats modules.')
+            ->addArgument(
+                'action',
+                InputArgument::OPTIONAL,
+                '( possible values : ' . $this->getPossibleActions() . ') ',
+                'status'
+            )
+            ->addOption(
+                'modulename',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Module to Install / Uninstall'
+            )
+            ->addOption(
+                'idsmodule',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'IDs module to Install or Uninstall separate by coma'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+        $helper = $this->getHelper('question');
+        $returnCode = null;
```

Comment:
```suggestion
```
$returnCode variable is never read, at multiple places.
you can just remove it.